jdi-light/src/test/java/io/github/epam/properties/DriversVersionTests.java

```diff
@@ -0,0 +1,44 @@
+package io.github.epam.properties;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import java.security.InvalidParameterException;
+import java.util.Properties;
+
+import static com.epam.jdi.light.common.Property.DRIVERS_VERSION;
+import static com.epam.jdi.light.common.PropertyValidationUtils.validateProperties;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.fail;
+
+public class DriversVersionTests {
+
+    @DataProvider
+    public static Object[] negativeData() {
+        return new Object[]{"latest", "LATES", "prelatest", "", "RELATEST", "2,31", "ten.eleven"};
+    }
+
+    @DataProvider
+    public static Object[] positiveData() {
+        return new Object[]{"LATEST", "1.23", "PRELATEST", "33.44"};
```

Comment:
Lets see valid values for this case: https://mvnrepository.com/artifact/org.seleniumhq.selenium/selenium-java
1. The third number always exists. Even if it's zero, so 1.23 and 33.44 are invalid, lets get rid of them in this test
2. Lets add some valid cases: 
- 4.0.0-alpha-3
- 3.141.59

Reply:
test data actualized, all tests are passed